core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -140,6 +141,12 @@ where
     }
 }
 
+impl ComponentMsgObj for WelcomeScreen {
```

Comment:
i think we don't need this impl when we don't put `WelcomeScreen` into a `LayoutObj` ?

Reply:
Deleted in [3562258](https://github.com/trezor/trezor-firmware/pull/2840/commits/356225877c2db956f6fb0dbe782b139e2cdb528e)